canvas_sdk/base.py

```diff
@@ -0,0 +1,45 @@
+from datetime import date, datetime
+from enum import Enum
+from typing import Any
+
+from pydantic import BaseModel, ConfigDict
+from pydantic_core import InitErrorDetails, PydanticCustomError, ValidationError
+
+
+class Model(BaseModel):
+    """A base model that includes validation methods."""
```

Comment:
I made both `DataModel` and `_BaseCommand` inherit from this class. It uses a lot of shared methods for validation.